src/pages/LandingPage.js

```diff
@@ -62,30 +51,27 @@ const makeCard = (link, title, body) => h(Clickable, {
   makeRightArrowWithBackgroundIcon()
 ])
 
-const getSiteSpecificHyperlinks = () => Utils.cond(
-  [isTerra(), () => [
-    makeDocLink('https://support.terra.bio/hc/en-us', 'Find how-to\'s, documentation, video tutorials, and discussion forums'),
-    makeDocLink('https://support.terra.bio/hc/en-us/articles/360033416672', 'Learn more about the Terra platform and our co-branded sites')
-  ]],
-  [isFirecloud(), () => [
-    makeDocLink('https://support.terra.bio/hc/en-us', 'Find how-to\'s, documentation, video tutorials, and discussion forums'),
-    makeDocLink('https://support.terra.bio/hc/en-us/articles/360022694271', 'Already a FireCloud user? Learn what\'s new.'),
-    makeDocLink('https://support.terra.bio/hc/en-us/articles/360033416912',
-      'Learn more about the Cancer Research Data Commons and other NCI Cloud Resources')
-  ]],
-  [isRareX(), () => [
-    makeDocLink('https://rare-x.org/DataAnalysisPlatform-Documentation',
-      'Find RARE-X Data Analysis Platform documentation, tutorials and Jupyter notebook examples'),
-    makeDocLink('https://rare-x.org/researchers/', 'Learn more about the RARE-X Data Analysis Platform'),
-    makeDocLink('https://support.terra.bio/hc/en-us', 'Find Terra how-to\'s, documentation, video tutorials, and discussion forums')
-  ]],
-  () => [makeDocLink('https://support.terra.bio/hc/en-us', 'Find how-to\'s, documentation, video tutorials, and discussion forums')]
+const makeDocLink = (href, title) => {
+  return div({ style: { marginBottom: '1rem', fontSize: 18 } }, [
+    h(Link, {
+      href,
+      ...Utils.newTabLinkProps,
+      style: { fontSize: 18 }
+    }, [
+      title,
+      icon('pop-out', { size: 18, style: { marginLeft: '0.5rem' } })
+    ])
+  ])
+}
+
+const makeDocLinks = linkTextPairs => _.map(
+  ({ link, text }) => makeDocLink(link, text), linkTextPairs
```

Comment:
Nit: If `makeDocLink` took one of the link configuration objects as a single parameter vs each value as separate parameters, this could be simplified a bit.

```js
const makeDocLink = ({ href, text }) => ...

const makeDocLinks = _.map(makeDocLink)
```

Reply:
Changed in [0f65cf6](https://github.com/DataBiosphere/terra-ui/pull/3257/commits/0f65cf6b943fcd45ebc65d53e4ba004743ea2ea6).